skeleton_scary_game.py

```diff
@@ -0,0 +1,329 @@
+import random
+
+
+# This is where the actual game will run from
+class Game_engine(object):
+
+    # This is the constructer method, this is what will be passed to the play() for the game to run
+    def __init__(self, demon, item, father, lore):
+        self.demon = demon
+        self.item = item
+        self.father = father
+        self.paths = ["Path 1", "Path 2", "Path 3"]
+        self.lore = lore
+
+        # This is the actual game where everything will run
+
+    def play(self):
+
+        # how many rounds you have survived,
+        loop_count = 0
+
+        while True:
+
+            # This will be used later if the user has a gun or not.
+            a = 0
+
+            # The path the demon will be on - randomizes every round
+            demon_path = random.choice(self.paths)
+
+            # Randomly assign the item path
+            item_paths = [path for path in self.paths if path != demon_path]
+            item_path = random.choice(item_paths)
+
+            # This is where the lore is
+            lore_paths = [path for path in self.paths if (path != demon_path and path not in item_path)]
+            lore_path = random.choice(lore_paths)
+
+            # Ask the user if they want to craft an item.
+            need_help = input(str("\nWould you like to craft an item? (y/n) -> "))
```

Comment:
Here might be a good place to add the help text on which items you need in your backpack to craft other ones.